devicon.json

```
@@ -6292,6 +6292,27 @@
             }
         ]
     },
+    {
+        "name": "renpy",
+        "altnames": [
+            "Ren'Py"
+        ],
+        "tags": [
+            "programming",
+            "language"
```

I'm not sure this fits for a programming language but more for a game engine.
```suggestion
            "programming",
            "game-engine",
            "engine",
            "python"
```

---

I'm not a user of `Ren'Py` nor `Python`, but I agree with you.
`Ren'Py` calls itself as a visual novel engine and looking through docs it's said:
> "Many portions of Ren'Py take Python expressions."

> "While Python expressions are very powerful, only a fraction of that power is necessary to write a basic Ren'Py game."

Looking through some `.rpy` files they look as a flavored `Python` to me, agreeing to the game engine idea.
That was my first contribution and now I got the general idea of those tags. Thank you!